src/main/resources/openapi/PrintAPIs.yaml

```diff
@@ -465,6 +465,100 @@ paths:
         connectionId: '${vpc_connection_id}'
         httpMethod: POST
 
+
+  #
+  # Anonymous Elector Document Summaries
+  # --------------------------------------------------------------------------------
+  '/eros/{eroId}/anonymous-elector-document-summaries':
```

Comment:
The problem with `anonymous-elector-document-summaries` is that it's not clear how it is different to the existing `anonymous-elector-documents` endpoint, especially considering the existing endpoint is document as returning "summaries"
I think we need to step back slightly and understand the terminology, resources, use cases etc; otherwise we are at risk of introducing at best a hard to grok API, and at worst a [Big Ball Of Mud](https://www.infoq.com/news/2014/08/microservices_ballmud/) :(

We need to slow down to go faster!!! (sorry, falling into Tim-style management speak!!! :D )